### PR TITLE
feat/#1439 Sentry Setup

### PR DESCRIPTION
**Pull request by badalkhatri0924**
https://github.com/ever-co/ever-teams/issues/1439

### CODE REVIEW

File: apps/web/sentry.client.config.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import * as Sentry from '@sentry/nextjs';
+
+Sentry.init({
+	dsn: process.env.NEXT_PUBLIC_SENTRY_DNS,
+
+	// Adjust this value in production, or use tracesSampler for greater control
+	tracesSampleRate: 1,
+
+	// Setting this option to true will print useful information to the console while you're setting up Sentry.
+	debug: false,
```
Comment (evereq) [inline apps/web/sentry.client.config.ts:14]: @badalkhatri0924 do we have ability to detect if it's dev or prod env? If yes, I would prefer to enable debug on DEV env and disable on any other!

File: apps/web/sentry.edge.config.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import * as Sentry from '@sentry/nextjs';
+
+Sentry.init({
+	dsn: process.env.NEXT_PUBLIC_SENTRY_DNS,
+
+	// Adjust this value in production, or use tracesSampler for greater control
+	tracesSampleRate: 1,
+
+	// Setting this option to true will print useful information to the console while you're setting up Sentry.
+	debug: false,
```
Comment (evereq) [inline apps/web/sentry.edge.config.ts:15]: @badalkhatri0924 same notice about DEV / PROD debug enable / disable

File: apps/web/sentry.server.config.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import * as Sentry from '@sentry/nextjs';
+
+Sentry.init({
+	dsn: process.env.NEXT_PUBLIC_SENTRY_DNS,
+
+	// Adjust this value in production, or use tracesSampler for greater control
+	tracesSampleRate: 1,
+
+	// Setting this option to true will print useful information to the console while you're setting up Sentry.
+	debug: false,
```
Comment (evereq) [inline apps/web/sentry.server.config.ts:14]: @badalkhatri0924 same as my other comments in this PR

**Comment by badalkhatri0924** [comment]
@evereq I've added optional NEXT_PUBLIC_SENTRY_DEBUG ENV

**Pull request merged**